website/index.html

```diff
@@ -21,87 +21,94 @@ <h2>
             </a>
         </header>
         <div class="flex-all-content">
+            <div class="gradient-for-line">
+            </div>
             <div class="line">
                 <div></div>
             </div>
-            <div id="main-content">
+            <div class="main-content">
                 <div class="text-image-flex">
                     <div class="introduction width-mobile">
                         <p>
                             <span>Hi! I am a developer</span>
-                            <br>
-                            <br>
+                            <br/>
+                            <br/>
                             I am a front end and 
-                            back end developer, that creates technical solutions for <br>
+                            back end developer, that creates technical solutions for <br/>
                             companies/clients of all genres.
-                            <br>
-                            <br>
+                            <br/>
+                            <br/>
                             Want to see some of my work? 
-                            <br>
+                            <br/>
                             ...You are looking at it!
                         </p>
                     </div>
                     <div class="image-container-3">
-                        <img src="image-1.png" alt="image could not be displayed">
+                        <img src="static/images/portrate.png" alt="Image could not be displayed"/>
                     </div>
                 </div>
                 <div class="text-image-flex">
                     <div class="development-for-multible-mediums width-mobile">
                         <p>
-                            <span>Development for multible mediums:</span><br>
-                            <br>
+                            <span>Development for multible mediums:</span>
+                            <br/>
+                            <br/>
                             <ul>
                                 <li>Web application development,</li>
                                 <li>Smartphone app development. </li>
                             </ul>
-                            <br>  
+                            <br/>  
                         </p>
                         <p>
-                            <span>Types of development:</span><br>
-                            <br>
+                            <span>Types of development:</span>
+                            <br/>
+                            <br/>
                             <ul>
                                 <li>Frontend & graphical user interface,</li>
                                 <li>Backend & server development.</li>
```

Comment:
No comma at the end of list items. Then also either always period at the end of list items/bulletpoints or never. Look at the list below - there u find no periods.
```suggestion
                                <li>Frontend & graphical user interface</li>
                                <li>Backend & server development</li>
```

Reply:
4883146